qiskit/transpiler/passes/post_mapping_optimization.py

```diff
@@ -0,0 +1,425 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+""" Methods to assist with compiling tasks.
```

Comment:
Is this a pass?

Reply:
If I understand correctly, these are utilities for the AStarCX pass. Maybe they should be methods in that object.